src/plugins/hathor_sqs.js

```diff
@@ -0,0 +1,74 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+/* istanbul ignore next */
+async function checkDeps() {
+  const requiredDeps = {
+    'aws-sdk': '^2.1226.0',
+    yargs: '^16.2.0',
+  };
+  await Promise.all(requiredDeps.map(async d => {
+    try {
+      return import(d);
+    } catch (e) {
+      throw new Error(`Some plugin dependencies are missing, to install them run:
+      $ npm install ${Object.entries(requiredDeps).map(x => [x[0], x[1]].join('@')).join(' ')}`);
+    }
+  }));
+}
+
+export function getSettings() {
+  const yargs = require('yargs/yargs'); // eslint-disable-line global-require
+  const { hideBin } = require('yargs/helpers'); // eslint-disable-line global-require
+  const { argv } = yargs(hideBin(process.argv));
+
+  const region = argv.plugin_sqs_region
+  || process.env.HEADLESS_PLUGIN_SQS_REGION
+  || null;
+  const queueUrl = argv.plugin_sqs_queue_url
+  || process.env.HEADLESS_PLUGIN_SQS_QUEUE_URL;
+  const endpoint = argv.plugin_sqs_endpoint_url
+  || process.env.HEADLESS_PLUGIN_SQS_ENDPOINT_URL
+  || null;
+
+  const sqsConfig = { apiVersion: '2012-11-05' };
+  if (endpoint !== null) {
+    sqsConfig.endpoint = endpoint;
+  }
+  if (region) {
+    sqsConfig.region = region;
+  }
+
+  return { region, queueUrl, endpoint, sqsConfig };
```

Comment:
We don't use `region` and `endpoint`. Why do we need them?

Reply:
Not really, we use them inside the `sqsConfig` but its just a wrapper for convenience.